ui/src/app/workflows/components/submit-workflow-panel.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import {Parameter, Workflow} from '../../../models';
+import {uiUrl} from '../../shared/base';
+import {ErrorNotice} from '../../shared/components/error-notice';
+import {services} from '../../shared/services';
+
+import {Select} from 'argo-ui';
+import {TagsInput} from '../../shared/components/tags-input/tags-input';
+
+interface Props {
+    kind: string;
+    namespace: string;
+    name: string;
+    entrypoint: string;
+    entrypoints: string[];
+    parameters: Parameter[];
```

Comment:
Why not simply pass in the `template`?

Reply:
because it is more information than I need - this makes it very clear what I need